lib/apollo-federation/tracing.rb

```diff
@@ -23,37 +15,10 @@ def should_add_traces(headers)
       headers && headers['apollo-federation-include-trace'] == KEY.to_s
     end
 
-    def attach_trace_to_result(result)
-      return result unless result.context[:tracing_enabled]
-
-      trace = result.context.namespace(KEY)
-      # TODO: If there are errors during query validation, that could also cause a missing
-      # start_time
```

Comment:
Is this issue addressed anywhere?

Reply:
Yeah, thats the main thing this is fixing. The start time is now recorded at the beginning of the `execute_multiplex` event, which will always happen before any parsing or validation. Also, now that the trace is added to the result automatically, we don't need to check if the tracer was properly installed (since the "add traces to result" code won't even run if the tracer isn't installed).